lib/all-badge-examples.js

```diff
@@ -726,7 +726,15 @@ const allBadgeExamples = [
       {
         title: 'Ansible Role',
         previewUri: '/ansible/role/d/3078.svg'
-      }
+      },
+      {
+        title: 'Launchpad.net PPA downloads',
```

Comment:
In my opinion world `downloads` is redundant here since whole section is named `Downloads`  (compare with http://shields.io/).